src/seedu/addressbook/data/person/DateOfBirth.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's date of birth in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDob(String)}
+ */
+public class DateOfBirth {
+
+    public static final String EXAMPLE = "12-09-1998";
+    public static final String MESSAGE_DOB_CONSTRAINTS = "Person dates of birth should be 3 numbers separated by '-'";
+    public static final String DOB_VALIDATION_REGEX = "\\d+-\\d+-\\d+";
```

Comment:
Try to see how you can improve this regex to handle months and day constraints.

Reply:
Is this a possible way? 
public static final String DOB_VALIDATION_REGEX = "\\d{2}-\\d{2}-\\d{4}";